lib/node_modules/@stdlib/math/base/special/rcbrt/README.md

```diff
@@ -0,0 +1,215 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2019 The Stdlib Authors.
```

Comment:
@steff456 Would you mind updating the copyright for all the relevant files in this PR to 2022?

Reply:
This has been done.